src/transports/abstract/subscription.ts

```diff
@@ -54,6 +59,11 @@ export abstract class SubscriptionTransport<const T extends TransportGenerics>
 
   async backgroundExecute(context: EndpointContext<T>): Promise<void> {
     logger.debug('Starting background execute')
+
+    if (this.retryTime > Date.now()) {
+      return
+    }
+
     const entries = await this.subscriptionSet.getAll()
```

Comment:
I'm curious about the motivations behind this change, any additional env vars are new risks and I'm wondering if this can be solved differently

Reply:
An alternative to this might be to set a specific retry limit per data provider, although exponential back-off was specifically requested and should help alleviate issues across all DPs.